docs/tutorial_bracket_strategy.md

```diff
@@ -1,62 +1,62 @@
----
-id: addtoken1
-title: Background
-sidebar_label: Background
----
+# Tutorial: Liquidity provision on Gnosis Protocol
```

Comment:
Maybe we also add the warning here: 
This summary is for informational purposes only. It is not a risk assessment, nor investment advice! It is not intended to substitute for the advice of licensed professionals. Gnosis believes that the information is accurate as of the date of publication. No warranty of accuracy is given, and no liability for any error or omission is accepted by Gnosis Ltd. and/or its affiliates.